### PR TITLE
[1.13.x][BXMSPROD-2057] Updating references to the new kie-ci actions folder

### PR DESCRIPTION
**Pull request by bacciotti**
Related PR:
- https://github.com/kiegroup/kogito-runtimes/pull/2965

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: .github/workflows/kogito-examples-pr.yml [hunk 1]
```diff
@@ -28,27 +28,27 @@ jobs:
     name: Maven Build
```
Comment (lampajr) [inline .github/workflows/kogito-examples-pr.yml:28]: As suggested here https://github.com/kiegroup/kogito-apps/pull/1755#pullrequestreview-1469203763 I would recommend switching to `@main` branch.

File: .github/workflows/kogito-runtimes-pr.yml [hunk 2]
```diff
@@ -28,26 +28,26 @@ jobs:
     name: Maven Build
```
Comment (lampajr) [inline .github/workflows/kogito-runtimes-pr.yml:28]: As suggested here https://github.com/kiegroup/kogito-apps/pull/1755#pullrequestreview-1469203763 I would recommend switching to `@main` branch.

**Review by lampajr — CHANGES_REQUESTED** [review]

File: .github/workflows/kogito-runtimes-pr.yml [hunk 3]
```diff
@@ -28,26 +28,26 @@ jobs:
...
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/kogito-pipelines/.ci/actions/build-chain@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@1.13.x
         env:
           BUILD_MVN_OPTS: '-Dproductized'
         with:
```
Comment (lampajr) [inline .github/workflows/kogito-runtimes-pr.yml:46]:
```suggestion
        with:
          definition-file: 'https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml'
```

File: .github/workflows/kogito-examples-pr.yml [hunk 4]
```diff
@@ -28,27 +28,27 @@ jobs:
...
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/kogito-pipelines/.ci/actions/build-chain@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@1.13.x
         env:
           BUILD_MVN_OPTS: '-Dproductized'
         with:
```
Comment (lampajr) [inline .github/workflows/kogito-examples-pr.yml:46]:
```suggestion
        with:
          definition-file: 'https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml'
```

**Review by lampajr — CHANGES_REQUESTED** [review]
Missing `definition-file`

**Review by lampajr — APPROVED** [review]

**Pull request merged**